packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -105,6 +126,10 @@
         },
         "when":
           "sfdx:project_opened && sfdx:replay_debugger_checkpoints_enabled"
+      },
+      {
+        "command": "sfdx.launch.replay.debugger.logfile",
```

Comment:
This command is **not** showing up in the palette right (just on right-click menu)?

If it is, then we need to check to ensure that if the user can invoke it, then we need more checks to see if there is an active editor and whether the contents of that active editor is a apexLog type.
It it is not, then we should be OK.

Reply:
I'll add some checks to make sure the text editor is apexlog type.